source/configure/rate-limiting-configuration-settings.rst

```diff
@@ -53,6 +53,8 @@ Maximum queries per second
   :environment: MM_RATELIMITSETTINGS_MAXBURST
   :description: The maximum number of requests allowed beyond the per second query limit when `rate limiting <#enable-rate-limiting>`__ is enabled. Default is **100** requests.
 
+  Increase this value to accept more requests each second, and decrease this value to allow fewer requests.
```

Comment:
@TomerPacific:

When you look at the generated preview, you'll see that the updated content doesn't display. That's because it's been added to the code section that drives search results, and not to the table details.

The final step of this PR is to move these details to the tables themselves (specifically to the left-hand description column). I'm happy to help with this step, or you're welcome to apply the needed changes yourself in this PR if you're open to it.